projects/stargazer/plugins/other/radius/server.cpp

```diff
@@ -48,10 +53,21 @@ RadProto::Packet Server::makeResponse(const RadProto::Packet& request)
     std::vector<uint8_t> vendorValue {0, 0, 0, 3};
     vendorSpecific.push_back(RadProto::VendorSpecific(m_dictionaries.vendorCode("Dlink"), m_dictionaries.vendorAttributeCode("Dlink", "Dlink-User-Level"), vendorValue));
 
-    if (request.type() == RadProto::ACCESS_REQUEST)
-        return RadProto::Packet(RadProto::ACCESS_ACCEPT, request.id(), request.auth(), attributes, vendorSpecific);
+    std::vector<RadProto::Attribute*> attributesEmpty;
+    std::vector<RadProto::VendorSpecific> vendorSpecificEmpty;
 
-    return RadProto::Packet(RadProto::ACCESS_REJECT, request.id(), request.auth(), attributes, vendorSpecific);
+    if (request.type() == RadProto::ACCESS_REQUEST)
+    {
+        if (findUser(request))
+            return RadProto::Packet(RadProto::ACCESS_ACCEPT, request.id(), request.auth(), attributes, vendorSpecific);
+        else
+        {
+            m_logger("Error findUser\n");
+            printfd(__FILE__, "Error findUser\n");
+            return RadProto::Packet(RadProto::ACCESS_REJECT, request.id(), request.auth(), attributesEmpty, vendorSpecificEmpty);
```

Comment:
If you need an empty vector, you don't have to create a dedicated variable. You can construct it in-place:
```c++
return RadProto::Packet(RadProto::ACCESS_REJECT, request.id(), request.auth(), {}, {});
```